nomad/conf/validators.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
I afford comments on minor stuff because I don't have much to comment on the rest. Why do you keep specifying source encoding in your units? it's utf-8 by default in python 3.